index.html

```diff
@@ -291,6 +291,125 @@ <h2>About me</h2>
+          <li class="full-name">
+            <label for="name"></label>
+            <input
+              class="text"
+              type="text"
+              id="name"
+              name="full_name"
+              placeholder="Full name"
+              minlength="3"
+              maxlength="30"
+              required
+            />
+            <img class="break" src="assets/images/Container.svg" alt="line" />
+          </li>
+
+          <li class="email">             
+            <label for="mail"></label>
+            <input
+              class="text"
+              type="email"
+              id="mail"
+              name="user_email"
+              placeholder="Email address"
+              required
+            />       
+
+            <img class="break" src="assets/images/Container.svg" alt="line" />
+          </li>
+
+          <li class="blog">
+            <label for="msg"></label>
+            <textarea
+              class="area"
+              id="msg"
+              name="user_message"
+              rows="10"
+              cols="50"
+              placeholder="Enter text here"
+              minlength="7"
+              maxlength="500"
+              required
+            ></textarea>
+          </li>
+
+          <li class="btn">
+            <button class="touch" type="submit">Get in touch</button>
+          </li>
+        </ul>
+        <div class="validation-msg">            
+        </div>    
+      </form>
+
+      <ul id="carte">
+        <li>
+          <a
+            href="https://github.com"             
+            rel="noopener noreferrer"
+          >
+            <img class="github" src="assets/images/git.png" alt="github logo"
+          /></a>
+        </li>
+        <li>
+          <a
+            href="https://www.vectorstock.com/"              
+            rel="noopener noreferrer"
+          >
+            <img
+              class="vector-one"
+              src="assets/images/i.png"
+              alt="vector-stock logo-one" />
+            <img
+              class="vector-two"
+              src="assets/images/n.png"
+              alt="vector-stock logo-two"
+          /></a>
+        </li>
+        <li>
+          <a
+            href="https://angel.co/"              
+            rel="noopener noreferrer"
+          >
+            <img
+              class="angel"
+              src="assets/images/ang.png"
+              alt="angellist logo"
+          /></a>
+        </li>
+        <li>
+          <a
+            href="https://twitter.com/"              
+            rel="noopener noreferrer"
+          >
+            <img
+              class="twitter"
+              src="assets/images/twit.png"
+              alt="twitter logo"
+          /></a>
+        </li>
+        <li>
+          <a
+            href="https://medium.com/"              
+            rel="noopener noreferrer"
+          >
+            <img class="medium" src="assets/images/med.png" alt="medium logo"
```

Comment:
- [ ] As mentioned by the previous reviewer, these images are not visible on the webpage. Please take a look at the screenshot taken from your web page below 👇 
![image](https://user-images.githubusercontent.com/90222110/175354345-cff61d93-c8a5-4dc6-a1d2-102b49d44146.png)

This is the common issue for developers in general and most of the case is due to either of the following reasons

- Image does not exist
- Image path stated in the `src` attribute is not correct

I think for your case, it is due to the image path which is not correct, so It would be better if you check it again and also try to refer to this [documentation](https://sebhastian.com/html-image-not-showing/) to understand how you can fix it 😺 🚀 
Remember to check all your images in your website 💡